__docs__/wonder-blocks-theming/tokens-border.mdx

```diff
@@ -0,0 +1,96 @@
+import {Meta, Source} from "@storybook/blocks";
+
+import TokenTable from "../../.storybook/components/token-table";
+
+import Banner from "@khanacademy/wonder-blocks-banner";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+<Meta title="Theming / Tokens / Border" />
+
+export const baseColumns = (kind) => [
+    {
+        label: "Token",
+        cell: (row) => (
+            <code>
+                border.{kind}.{row.label}
+            </code>
+        ),
+    },
+    {
+        label: "Value",
+        cell: "value",
+    },
+];
+
+export const mapFunction = ([key, value]) => ({
```

Comment:
It looks like this map function is rewritten/used in `tokens-color.mdx`, `tokens-font.mdx`, and `tokens-spacing.mdx`. What if you move this into `token-table.tsx` or maybe some other util file and export it? And then it can be imported in all these mdx files?

Reply:
yeah good point! I think I'm just going to change the shape of data, so we can call those functions internally and we just pass the tokens object directly.